src/backend/metal/Cargo.toml

```diff
@@ -33,4 +33,4 @@ libc = "0.2"
 objc = "0.1.8"
 objc-foundation = "0.1"
 bit-set = "0.4"
-metal-rs = "0.1"
+metal-rs = { git = "https://github.com/fkaa/metal-rs", rev = "5263898c2ce1fb8263653272014366eba45ef88d" }
```

Comment:
That revision includes the merging of a pull request I recently submitted (to metal-rs) to add the binding for the newLibraryFromFile method.

I think it might be better to hold off because I plan to keep working on the metal backend which may require some more additions to the bindings, and the metal backend is not on crates.io itself at the moment, but that works too.